src/test/views/ResquestPasswordChange.test.tsx

```diff
@@ -0,0 +1,141 @@
+import { render, screen, fireEvent, waitFor, act } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import RequestPasswordChange from '../../views/RequestPasswordChange';
+import axiosClient from '../../hooks/AxiosInstance';
+import "@testing-library/jest-dom";
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useLocation: () => ({ state: { email: 'test@example.com' } }),
+}));
+
+jest.mock('../../hooks/AxiosInstance.tsx');
+jest.mock('react-toastify');
+
+describe('RequestPasswordChange Component', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
I think it can be better if these files are placed in tests folder

Reply:
They are in test/views directory